Prepare 3.16.0 (#1270)
conda/constructor: CHANGELOG.md

[//]: # (current developments)

## 2026-06-22   3.16.0:
### Enhancements

> * EXE: Uninstalling with option "Clear Package Cache" now also performs additional clean-up to account for potentially left-over files. (#1213 via #1214)

Reviewer: ```suggestion
* EXE: Uninstalling with option "Clear Package Cache" now also performs additional clean-up to account for potentially left over files. (#1213 via #1214)
```

> * Add `installer_type: docker` support to generate a Dockerfile and staged `.sh` installer from a constructor build to be used as-is or customized before building.

Reviewer: ```suggestion
* Add `installer_type: docker` support to generate a Dockerfile and staged `.sh` installer from a constructor build (#1219).
```

> * Add `docker_image_format: tar` support to build and export a portable Docker image. Requires Docker Buildx.

Reviewer: ```suggestion
* Add `docker_image_format: tar` support to build and export a portable Docker image. Requires Docker Buildx (#1219).
```

> * Both Docker features target Linux only and support all host platforms. (#1219)

Reviewer: ```suggestion
```

This isn't a new feature, so I would remove that line.

> * Both Docker features target Linux only and support all host platforms. (#1219)

Reviewer: ```suggestion
```
